microsoft-edge/web-platform/release-notes/136.md

```diff
@@ -0,0 +1,498 @@
+
+See [Intl.Locale](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale) at MDN.
+
+
+<!-- ====================================================================== -->
+## Origin trials
+
+The following are new experimental APIs that you can try on your own live website for a limited time.
+
+To learn more about origin trials, see [Use origin trials in Microsoft Edge](../../origin-trials/index.md).
+
+To see the full list of available origin trials, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials/).
+
+
+<!-- ------------------------------ -->
+#### New Chromium origin trials
+
+
+<!-- ---------- -->
+###### Update of Canvas text rendering implementation
+
+Expires on December 16, 2025.
+
+New implementation of `measureText()`, `fillText()`, and `strokeText()` for the CanvasRenderingContext2D API.  You can compare the performance of these methods against the previous implementation of these methods.
+
+* [Explainer](https://docs.google.com/document/d/1UbHuoflNyLbSavpH86HEmXFGyz2V4uZ6cjNpuuwrtBg/edit?usp=sharing)
+* [Feedback](https://crbug.com/new)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/3227392082964381697)
+
+
+<!-- ---------- -->
+###### Set preferred audio output device
+
+Expires on January 27, 2026.
+
+This feature allows a page to set a non-default device for all audio output from the page, including its child frames.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/SetPreferredSinkId/explainer.md)
+* [Feedback](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/1173750652883435521)
+
+
+<!-- ---------- -->
+###### `confidence` field on `PerformanceNavigationTiming` object
+
+Expires on October 14, 2025.
+
+The `PerformanceNavigationTiming` object now includes a `confidence` field.  Use the `confidence` field to evaluate whether navigation timings accurately reflect your web app's performance.  The data initially contains some noise, because this field employs differential privacy, to safeguard user privacy.  By gathering this field in your Real User Monitoring (RUM) data, you can filter out the noise with enough records to analyze trends and gain meaningful insights over time.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PerformanceNavigationTiming%20for%20User%20Agent%20Launch/explainer.md)
+* [Feedback](https://issues.chromium.org/issues/new?component=1456492&priority=P2&type=bug&noWizard=true&title=Feedback%20for%20Confidence%20field&cc=mjackson@microsoft.com&template=0)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/-8409909354160979967)
+
+
+<!-- ------------------------------ -->
+#### Microsoft Edge-only origin trials
+
+
+<!-- ---------- -->
+###### `handwriting` attribute
```

Comment:
We should remove this. It as deprecated earlier today. Sorry for the late catch.